city_scrapers/spiders/il_medicaid.py

```diff
@@ -0,0 +1,95 @@
+from city_scrapers_core.constants import ADVISORY_COMMITTEE
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from datetime import datetime
+
+class IlMedicaidSpider(CityScrapersSpider):
+    name = "il_medicaid"
+    agency = "Illinois Medical Adivsory Committee"
+    timezone = "America/Chicago"
+    allowed_domains = ["www.illinois.gov"]
+    # start_urls = ["file:///Users/ksong/myCodes/open-source/city-scrapers/tests/files/il_medicaid.html"]
+    start_urls = ["https://www.illinois.gov/hfs/About/BoardsandCommisions/MAC/Pages/"
+                  "MeetingSchedule.aspx"]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+
+        date_x_path = '//div[@id="ctl00_PlaceHolderMain_ctl01_ctl01__ControlWrapper_RichHtmlField"]/ul/li/p/text()'
+        for raw_date in response.xpath(date_x_path).getall():
+            # Clean up the raw dates
+            raw_date = raw_date.replace('\xa0',' ').strip()
+            if raw_date.count(',') == 2:
+                index = raw_date.find(",")
+                raw_date = raw_date[index+1:].strip()
+            date = datetime.strptime(raw_date, "%B %d, %Y")
+
+            meeting = Meeting(
+                title='Medicare Advisory Committee',
+                description='',
+                classification=ADVISORY_COMMITTEE,
+                start=date.replace(hour=10),
```

Comment:
That's fine by me. I was under the impression from the [contribution guide](https://cityscrapers.org/docs/development/#a-write-parse-methods-in-the-spider) that this was the preference:

> Often a value for meetings returned by a spider will be the same regardless of meeting content (an example is that most meetings will always have False for the all_day value). In this case, feel free to remove the _parse_* method for that field, and simply include the value in each dictionary (so 'all_day': False in this example rather than 'all_day': self._parse_all_day(item)).

Perhaps this part of the documentation should be modified with some examples?
 should be updated, perhaps with some examples:

Reply:
Thanks for clarifying. We should definitely specify there that it mostly applies for things like `classification`, `all_day`, and sometimes `name` but that scheduling details like time and location should be pulled from the page if available

If you're interested, feel free to open a PR with clearer language. Otherwise I can make that update

Reply:
Opened #702